src/Symfony/Component/DependencyInjection/Tests/Compiler/ValidateEnvPlaceholdersPassTest.php

```diff
@@ -105,7 +105,6 @@ public function testInvalidEnvInConfig()
     public function testNulledEnvInConfig()
     {
         $this->expectException('Symfony\Component\Config\Definition\Exception\InvalidTypeException');
-        $this->expectExceptionMessage('Invalid type for path "env_extension.int_node". Expected int, but got NULL.');
```

Comment:
Can't we keep it and relax the message we expect (with placehoders)?

Reply:
updated